compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.td

```diff
@@ -243,6 +243,12 @@ def AMDAIEFlattenLogicalObjectFifo :
   let constructor = "mlir::iree_compiler::AMDAIE::createAMDAIEFlattenLogicalObjectFifoPass()";
 }
 
+def AMDAIEFunctionOutlining :
```

Comment:
Add more info like what the conditions are for outlining, i.e. it happens on linalg ops.
Also, I think `AMDAIELinalgFunctionOutlining` would be a more descriptive name.